prawns/staking/src/handlers/staking.action.event.ts

```diff
@@ -33,6 +27,10 @@ export default (network: SubstrateNetwork, tokenIndex: number, action: Substrate
         data.amount = stakingBondedEvent.amount;
         data.stash = await getOrCreateStash(ctx, stakingBondedEvent.stash, symbol, network);
 
+        if (ctx.extrinsic) {
```

Comment:
maybe put an else throw Error in here, in theory it should never throw... I've been doing this more in the EVM work when we know something should be there but for whatever reason the types can't guarantee it

Reply:
Makes sense, I'll include it 👍